src/backend/Aggregator/HttpServices/Interceptors/HttpClientAuthorizationDelegatingHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Authentication;
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+using System.Collections.Generic;
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace HttpServices.Interceptors
+{
+    public class HttpClientAuthorizationDelegatingHandler : DelegatingHandler
+    {
+        private readonly IHttpContextAccessor _httpContextAccesor;
+
+        public HttpClientAuthorizationDelegatingHandler(IHttpContextAccessor httpContextAccesor)
+        {
+            _httpContextAccesor = httpContextAccesor;
+        }
+
+        protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+        {
+            string authorizationHeader = _httpContextAccesor
+                .HttpContext
+                .Request
+                .Headers["Authorization"];
+
+            if (!string.IsNullOrEmpty(authorizationHeader))
+            {
+                request.Headers.Add("Authorization", new List<string>() { authorizationHeader });
+            }
+
+            string token = await GetToken();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5020178)